### PR TITLE
Improve documentation for SpritesheetLoader

### LINKED ISSUE #6851: Please document spritesheetLoader behaviour
Subj

https://github.com/pixijs/pixi-tilemap/issues/11

### PR DESCRIPTION
**Pull request by bigtimebuddy**
Fixes #6851

http://pixijs.download/fix-6851/docs/PIXI.SpritesheetLoader.html

### CODE REVIEW

**Review by themoonrat — APPROVED** [review]

File: packages/spritesheet/src/SpritesheetLoader.ts [hunk 1]
```diff
@@ -9,6 +9,20 @@ import type { Loader, ILoaderResource } from '@pixi/loaders';
  *
  * This middleware automatically generates Texture resources.
  *
+ * If you're using Webpack or other bundlers and plan on bundling the atlas' JSON,
+ * use the {@link PIXI.Spritesheet Spritesheet} class to directly parse the JSON.
```
Comment (bigtimebuddy) [inline packages/spritesheet/src/SpritesheetLoader.ts:13]: @SukantPal I noticed that the `@link` tags here don't work. Is this a known issue in Webdocs

Comment (SukantPal) [reply packages/spritesheet/src/SpritesheetLoader.ts:13]: @bigtimebuddy Use `{@link PIXI.Spritesheet}`. I think (not sure tho) `[Spritesheet]{@link PIXI.Spritesheet}` might work as well.

Comment (bigtimebuddy) [reply packages/spritesheet/src/SpritesheetLoader.ts:13]: Here are the JSDoc formats. Is it possible to support the last two (sounds like they aren't supported yet)? I think all these formats are used in our docs.
```
{@link namepathOrURL}
[link text]{@link namepathOrURL}
{@link namepathOrURL|link text}
{@link namepathOrURL link text (after the first space)}
```

Comment (SukantPal) [reply packages/spritesheet/src/SpritesheetLoader.ts:13]: The first two should be working right now. I'll work on adding the other two.

**Review by GoodBoyDigital — APPROVED** [review]
👍

**Review by SukantPal — APPROVED** [review]

**Pull request merged**